com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/ICompletionsProvider.java

```diff
@@ -0,0 +1,35 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Microsoft Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Microsoft Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.microsoft.java.debug.core.adapter;
+
+import java.util.List;
+
+import com.microsoft.java.debug.core.protocol.Types.CompletionItem;
+import com.sun.jdi.StackFrame;
+
+public interface ICompletionsProvider extends IProvider {
```

Comment:
Where is the implementation of `ICompletionsProvider`?

Reply:
In another part of code review.